api/pkg/apis/v1alpha1/utils/apiclient.go

```diff
@@ -560,6 +561,27 @@ func (a *apiClient) PublishActivationEvent(ctx context.Context, event v1alpha2.A
 	return nil
 }
 
+func (a *apiClient) GetActivation(ctx context.Context, activation string, namespace string, user string, password string) (model.ActivationState, error) {
+	ret := model.ActivationState{}
+	token, err := a.tokenProvider(ctx, a.baseUrl, a.client, user, password)
+
+	if err != nil {
+		return ret, err
+	}
+
+	response, err := a.callRestAPI(ctx, "activations/registry/"+url.QueryEscape(activation)+"?namespace="+url.QueryEscape(namespace), "GET", nil, token)
```

Comment:
Handle the case when namespace is empty string

Reply:
We can discuss with our team

Reply:
#466